packages/compass-explain-plan/src/components/explain-json/explain-json.module.less

```diff
@@ -6,10 +6,10 @@
   .document-list {
     padding: 0;
     position: relative;
-    background: white;
-    border: 1px solid #ddd;
+    background: #ffffff;
+    border: 1px solid #dddddd;
     border-radius: 4px;
-    box-shadow: 0 1px 1px rgb(0 0 0 / 5%);
```

Comment:
I never know you could do something like this, I don't think I would though.

Reply:
yeah ... that was new for me too! that's when I discovered that VScode can actually convert between any color format by clicking on the color box, saved me 1 hour of copy/paste

<img width="406" alt="Screenshot 2022-10-06 at 10 02 20" src="https://user-images.githubusercontent.com/334881/194254441-dd465efe-b947-4145-9f0a-23ffb3c1d368.png">